src/MODS/showpopup.js

```diff
@@ -0,0 +1,139 @@
+
+  const showtype = document.createElement('li');
+  showtype.className = 'stats';
+  showtype.innerText = `Type: ${type}`;
+  showstats.appendChild(showtype);
+
+  const showlanguage = document.createElement('li');
+  showlanguage.className = 'stats';
+  showlanguage.innerText = `Language: ${language}`;
+  showstats.appendChild(showlanguage);
+
+  const showgenres = document.createElement('li');
+  showgenres.className = 'stats';
+  showgenres.innerText = `Genres: ${genres}`;
+  showstats.appendChild(showgenres);
+
+  const showrating = document.createElement('li');
+  showrating.className = 'stats';
+  showrating.innerText = `Star-rating: ${rating}`;
+  showstats.appendChild(showrating);
+
+  commentsPopup.appendChild(showstats);
+
+  // people comments .......
+
+  //    //comments heading
+  //     const commenthd = document.createElement("h2");
+  //     commenthd.className = "hcomments";
+  //     commenthd.innerText = "Viewer's comments";
+  //     commentsPopup.appendChild(commenthd);
+
+  //  // given comments
+  //     const viewers = document.createElement("ul");
+  //     viewers.className = "people-comments";
+
+  // //temporary viewer list
+  //     const viewerA= document.createElement("li");
+  //     viewerA.className = "stats";
+  //     viewers.appendChild(viewerA);
+
+  //     const viewerB= document.createElement("li");
+  //     viewerB.className = "viewer";
+  //     viewers.appendChild(viewerB);
+
+  //     commentsPopup.appendChild(viewers);
+
+  // //FORM ADD VIEWER AND COMMENT
+  //     //main container
+  //         const formsec = document.createElement("section");
+  //         formsec.className = "addComments";
+  //         commentsPopup.appendChild(formsec);
+
+  //     //heading of the section
+  //         const hAdd = document.createElement("h2");
+  //         hAdd.className = "hAdd";
+  //         hAdd.innerText = "Add a comment";
+  //         formsec.appendChild(hAdd);
+
+  //     //FORM .......................
+  //         const formmain = document.createElement("form");
+  //         formmain.action = "#" ;
+  //         formmain.method = "GET" ;
+  //         formsec.appendChild(formmain);
+
+  // //inside form inputs
+  // const user = document.createElement("input");
+  // user.className = "text";
+  // user.type = "text" ;
+  // user.id = "username";
+  // user.required = true;
+  // user.placeholder = "Enter your name";
+  // formmain.appendChild(user);
+
+  // const usercomm = document.createElement("input");
+  // usercomm.className = "text";
+  // usercomm.type = "text" ;
+  // usercomm.id = "comment";
+  // usercomm.required = true;
+  // usercomm.placeholder = "Your insights";
+  // formmain.appendChild(usercomm);
+
+  // const commBtn = document.createElement("input");
+  // commBtn.className = "com-btn";
+  // commBtn.type = "submit" ;
+  // commBtn.id = "submit";
+  // commBtn.value = "Comment";
+  //  formmain.appendChild(usercomm);
+
+  body.appendChild(commentsPopup);
+};
```

Comment:
- [ ] Kindly remove the code commented if it is not going to be used anymore. If you are going to give future use to the lines, please omit this request.